mcs_benchmark_data/models/benchmark_dataset.py

```diff
@@ -11,4 +15,17 @@ class BenchmarkDataset(_Model):
     '''A file containing elements (questions, answers, context, observations, ...) of a benchmark'''
     '''Sub-classes: BenchmarkDevDataset, BenchmarkTestDataset, BenchmarkTrainingDataset'''
     name: str
-    entries: Tuple[BenchmarkSample,...]
\ No newline at end of file
+    entries: Tuple[BenchmarkSample,...]
+
+    def to_rdf(
+        self, *, graph: Graph, **kwds) -> Resource:
+        resource = _Model.to_rdf(
+            self, graph=graph, **kwds
+        )
+        resource.add(RDF.type, MCS[self.__class__.__name__])

```

Comment:
You're duplicating this code in various places, just add it to _Model.to_rdf if it's universal?